app/models/pglogical_subscription.rb

```diff
@@ -101,15 +100,24 @@ def self.provider_node_attributes(node_name)
 
   private
 
-  def node_name
-    MiqPglogical::NODE_PREFIX + MiqRegion.my_region_number.to_s
+  def update_subscription
+    disable
+    provider_node_name = MiqPglogical.region_to_node_name(provider_region)
+    pglogical.node_dsn_update(provider_node_name, dsn)
+    enable
+  end
+
+  def create_subscription
+    create_node.check if !pglogical.enabled? || self.class.count == 0
```

Comment:
Maybe create_node can just be resilient and just return if it's already defined?